support/support.go

```diff
@@ -329,6 +333,28 @@ func AgentInfo(ctx context.Context, client *codersdk.Client, log slog.Logger, ag
 		if !conn.AwaitReachable(ctx) {
 			log.Error(ctx, "timed out waiting for agent")
 		} else {
```

Comment:
It might be time to re-think this ifelse logic 😄. Maybe break into functions or consider early return?

Reply:
Early return gets a bit messy; I may just break into a function.

Reply:
Yea, function or handling in defer would probably work best.